src/layout.rs

```diff
@@ -432,6 +458,7 @@ impl<const C: usize, const R: usize, const L: usize, T: 'static> Layout<C, R, L,
                 }
             }
         }
+        self.tap_hold_tracker.coord = coord;
```

Comment:
I did it this way since it had the smallest amount of code added and thought that only branch that doesn't work correctly is `CustomEvent`, since `MultipleActions` recurses. I've changed it in the latest force push by putting it in every branch.